extensions/azdata/package.nls.json

```diff
@@ -2,5 +2,23 @@
 	"azdata.displayName": "Azure Data CLI",
 	"azdata.description": "Support for Azure Data CLI.",
 	"azdata.config.title": "Azure Data CLI Configuration",
-	"azdata.config.debug": "Log debug info to the output channel for all executed azdata commands"
+	"azdata.config.debug": "Log debug info to the output channel for all executed azdata commands",
+
+	"command.category": "Azure Data CLI",
+	"azdata.acceptEula.command.name": "Accept Eula",
+	"azdata.install.command.name": "Install",
+	"azdata.upgrade.command.name": "Check for Upgrade",
```

Comment:
`Check for updates` - that's the pretty standard string used for this type of thing.